### ISSUE TITLE
DITA 1.3 grammar files not shipping the final version

### ISSUE THREAD

**Issue by robander** [dita-ot/dita-ot#2401]
DITA-OT 2.3 is shipping the "Candidate OASIS Standard" copy of grammar files, rather than the final OASIS standard copies. We should ship the final copies instead.

**Comment by rmraya** [comment]
The official XML catalog for DITA 1.3 has errors.  Some definitions are missing. 

I use these entries in the catalogs deployed with my tools:


		

	<nextCatalog catalog="dita1.3/catalog.xml"/>  	  	
	
	
	
   	<public publicId="-//OASIS//ENTITIES DITA 1.3 Topic Definitions//EN"  uri="dita1.3/dtd/base/dtd/topicDefn.ent"/>
	<public publicId="-//OASIS//ENTITIES DITA Delivery Target Domain//EN" uri="dita1.3/dtd/base/dtd/deliveryTargetAttDomain.ent"/>
	<public publicId="-//OASIS//ELEMENTS DITA Machinery Taskbody Constraint//EN" uri="dita1.3/dtd/machineryIndustry/dtd/machineryTaskbodyConstraint.mod"/>
	<public publicId="-//OASIS//ELEMENTS DITA Map Subject Classification Domain//EN" uri="dita1.3/dtd/subjectScheme/dtd/classifyDomain.mod"/>
	<public publicId="-//W3C//ENTITIES MathML 3.0 Qualified Names 1.0//EN" uri="dita1.3/dtd/technicalContent/dtd/mathml/mathml3/mathml3-qname.mod"/>

Please consider including similar fixes in the OT's catalog if you ship the official grammars.

**Comment by robander** [comment]
Catalog fixes went into 2.2.5 (issue #2266) and were carried forward into 2.3 -- that fix should cover all of the catalog issues from the original 2.3 delivery.

**Comment by robander** [comment]
Commit 85e3bdb updates to latest versions of the grammar files. This hit a lot of files but only updates metadata (showing version of the files and download link).

I'm keeping the catalog issue separate - will take a look at that now and either update, or mark this closed if we've got everything correct.

**Comment by robander** [comment]
Fixed in 2.2.5:
```
<public publicId="-//OASIS//ENTITIES DITA 1.3 Topic Definitions//EN"  uri="dita1.3/dtd/base/dtd/topicDefn.ent"/>
```

The following is invalid, the correct public ID is` "-//OASIS//ENTITIES DITA 1.3 Delivery Target Attribute Domain//EN"`:
```
<public publicId="-//OASIS//ENTITIES DITA Delivery Target Domain//EN" uri="dita1.3/dtd/base/dtd/deliveryTargetAttDomain.ent"/>
```

The following is also invalid, the correct public ID is `"-//OASIS//ELEMENTS DITA Taskbody Constraint//EN"`:
```
<public publicId="-//OASIS//ELEMENTS DITA Machinery Taskbody Constraint//EN" uri="dita1.3/dtd/machineryIndustry/dtd/machineryTaskbodyConstraint.mod"/>
```

The following is also invalid, correct public ID is `""-//OASIS//ELEMENTS DITA Subject Classification Domain//EN"`:

```
<public publicId="-//OASIS//ELEMENTS DITA Map Subject Classification Domain//EN" uri="dita1.3/dtd/subjectScheme/dtd/classifyDomain.mod"/>
```

The last one is a module within MathML. We do have a catalog entry for the main MathML DTD file, which then pulls in the `mathml3-qname.mod` file. I'm not sure if this was intentionally left out of the OASIS catalogs or not. @drmacro - any thought on whether this should be part of the OASIS delivery (I ask, realizing that OT delivery is a slightly different question, but I'd like to keep them in sync)?
```
<public publicId="-//W3C//ENTITIES MathML 3.0 Qualified Names 1.0//EN" uri="dita1.3/dtd/technicalContent/dtd/mathml/mathml3/mathml3-qname.mod"/>
```

**Comment by drmacro** [comment]
The MathML .mod file is pulled in by direct URI reference to avoid having the entity reference redirected to a different version of that file. Many environments have multiple and incompatible versions of the MathML DTDs around. This approach ensures that the DITA-specific configuration is used.

**Comment by robander** [comment]
Backing up a bit. 

I listed 3 public IDs as invalid, but I'm suddenly worried about two of them.

- Delivery Target - that is new to DITA 1.3, so whatever shipped with 1.3 is the proper one. So, I still consider this invalid.
- Machinery Taskbody Constraint - the public ID that @rmraya showed was the one shipped as the "Version Agnostic" one in DITA 1.2. So I think this should have been the same shipped with 1.3. Instead it appears we changed the public ID.
- Subject Classification domain - same issue. The one listed above is the one we had in DITA 1.2 catalogs. The name changed in 1.3.

**Comment by robander** [comment]
I've updated catalogs to add the DITA 1.2 version-agnostic public ID for the machinery constraint and for the classification domain. OASIS will be restoring each of these to the catalog in DITA 1.3 errata.

With that I'm marking this closed. 

@infotexture draft for release notes: 
> The DITA 1.3 DTD, Schema, and RNG files have been updated to the final "OASIS Standard" copy; this change updates the OASIS link and date in many header files, but does not affect how the grammar files work. In addition, the catalog has been updated to restore two entries that will eventually be shipped in the DITA 1.3 Errata from OASIS.
